### DISCUSSION TITLE
Add keybinding send C-x C-e to C-x C-e when C-x as leader exists

### DISCUSSION THREAD

**Discussion by zw963** [wezterm/wezterm#5030] [General]
Following is my config.

```
config.leader = { key = 'x', mods = 'CTRL' }
config.keys = {
  {
    key = '3',
    mods = 'LEADER',
    action = act.SplitHorizontal { domain = 'CurrentPaneDomain' },
  },
  {
    key = '2',
    mods = 'LEADER',
    action = act.SplitVertical { domain = 'CurrentPaneDomain' },
  }
}
```
When this config take effect, i can't run `C-x C-e` to invoke my editor for edit.

How to rebind C-x C-e to  send key `C-x C-e`?

Thanks.

**Comment by wez** [comment]
See the example in the docs: https://wezfurlong.org/wezterm/config/keys.html#leader-key

**Comment by zw963** [comment]
Okay, i solve this issue partially, the only left question is, how to send `C-x C-e` use action.SendKey ?

following is a example:

```lua
config.leader = { key = 'x', mods = 'CTRL' }

config.keys = {
  {
    key = 'e',
    mods = 'LEADER|CTRL',
    action = act.SendKey {
      -- how to send C-x C-e here?
    }
  },
}
```

What i expected is when i pressing `C-x C-e` (above code work on this aspect), how to send `C-x C-e` use SendKey?

Thanks.

**Comment by zw963** [comment]
Yeah! i solved my issue completely!

following is config:

```lua
  {
    key = 'e',
    mods = 'LEADER|CTRL',
    action = act.Multiple {
      act.SendKey { key = 'x', mods = 'CTRL' },
      act.SendKey { key = 'e', mods = 'CTRL' }
    }
  },
```

Thank you very much!
